doc/classes/Environment.xml

```diff
@@ -9,6 +9,7 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
+		These effects will only apply when the Viewport usage is 3D.
```

Comment:
"when the Intended usage in project settings is set to 3D" or "Viewport usage in project setting is set to 3d"